cmd/wksctl/init/init.go

```diff
@@ -109,7 +117,12 @@ func updatedArg(item string) []byte {
 }
 
 func updateControllerManifests(contents []byte, options initOptionType) ([]byte, error) {
-	return controllerImageSegment.ReplaceAll(contents, []byte(`$1:`+options.version)), nil
+	withVersion := controllerImageSegment.ReplaceAll(contents, []byte(`$1:`+options.version))
+	if controllerFootlooseEnvEntry.Find(withVersion) == nil {
+		return controllerFootlooseAddrLocation.ReplaceAll(withVersion,
```

Comment:
I think a comment here about this string formatting specifics would be worthwhile.